source/renderer/app/containers/wallet/dialogs/send-confirmation/Container.tsx

```diff
@@ -0,0 +1,122 @@
+import React, { useMemo, useCallback, useState } from 'react';
+import { inject, observer } from 'mobx-react';
+import compose from 'lodash/fp/compose';
+import { getNonZeroAssetTokens } from '../../../../utils/assets';
+import { ContainerProps as Props, SubmitPayload } from './types';
+import { WalletSendConfirmationDialogView } from './View';
+
+export const Containter = ({
+  actions,
+  stores,
+  amount,
+  selectedAssets,
+  assetsAmounts,
+  receiver,
+  totalAmount,
+  onExternalLinkClick,
+  transactionFee,
+  hwDeviceStatus,
+  isHardwareWallet,
+  formattedTotalAmount,
+}: Props) => {
+  // @ts-ignore ts-migrate(2339) FIXME: Property 'isFlight' does not exist on type 'typeof... Remove this comment to see the full error message
+  const { isFlight } = global;
```

Comment:
you could move the declaration to `source/renderer/declaration.d.ts`

Reply:
Updated, thanks!